src/replica/replica_2pc.cpp

```diff
@@ -116,6 +117,21 @@ DSN_DEFINE_uint64(
 DSN_DECLARE_int32(max_mutation_count_in_prepare_list);
 DSN_DECLARE_int32(staleness_for_commit);
 
+namespace {
+error_code disk_status_to_error_code(disk_status::type ds)
+{
+    switch (ds) {
+    case disk_status::SPACE_INSUFFICIENT:
+        return dsn::ERR_DISK_INSUFFICIENT;
+    case disk_status::IO_ERROR:
+        return dsn::ERR_DISK_IO_ERROR;
+    default:
```

Comment:
It's better to move `disk_status::NORMAL` in front of all other cases to improve cache efficiency

Reply:
Now the only caller will not pass a disk_status::NORMAL parameter to this function.